bin/dataduct

```diff
@@ -170,96 +217,158 @@ def main():
     )
     subparsers = parser.add_subparsers(
         dest='command',
-        help='Actions for various features')
+        help='Actions for various features',
+    )
 
-    # Config parser declaration
-    config_choices = {
-        CONFIG_TO_S3: 'sync config file from local to s3',
-        CONFIG_FROM_S3: 'sync config file from s3 to local file',
-    }
-    config_parser = subparsers.add_parser(
-        CONFIG_COMMAND,
-        parents=[base_parser],
-        formatter_class=formatter_class,
-        help='Command to sync config to and from S3'
-    )
-    config_parser.add_argument(
-        'action',
-        type=str,
-        choices=config_choices,
-        help=choice_help_formatter(config_choices),
-        default=CONFIG_FROM_S3,
-    )
-
-    # Pipeline parser declaration
-    pipeline_choices = {
-        CREATE_STR: 'Create a pipeline locally',
-        VALIDATE_STR: 'Validate a pipeline with AWS without activating',
-        ACTIVATE_STR: 'Activate the pipeline on AWS',
-        VISUALIZE_STR: 'Visualize the pipeline',
-    }
+    # Pipeline parser
```

Comment:
I don't know how much of the code is shared as we change parents for some and the command obviously. It seems like we would create un-needed wrapper.